packages/workbox-cli/src/lib/errors.js

```diff
@@ -24,4 +24,14 @@ module.exports = {
   'config-validation-failed': `Your configuration is invalid:`,
   'workbox-build-runtime-error': `Service worker generation failed:`,
   'unknown-command': `Unknown command:`,
+  'no-file-extensions-found': ol`No files could be found that are suitable for
+    caching.`,
+  'no-file-extensions-selected': `Please select at least one file extension.`,
+  'sw-src-does-not-exist': ol`The source service worker file you provided does
```

Comment:
Nit: Feels a little awkward "The source service worker file you provided does not exist. Please provide the path to an existing file."

Maybe change to something like:

"Could not find the service worker file at '<Insert absolute file path here>'. Please make sure it exists."

Reply:
Actually, turns out that error string isn't being used—I think I carried it over from v2? I've deleted it.